test/main.cpp

```diff
@@ -0,0 +1,13 @@
+#include <catch2/catch_test_macros.hpp>
```

Comment:
This might be out of the scope of this MR, but if you could write a real test, then all of the file dependencies will be finished. That way, the rest of us to just add more tests.

Reply:
I think I can do this in another PR. This would also address https://github.com/thedavidchu/merry_mem/pull/4#discussion_r1382596475.